proto/engine/v1/json_marshal_unmarshal.go

```diff
@@ -499,11 +569,31 @@ func (p *PayloadAttributes) MarshalJSON() ([]byte, error) {
 
 // MarshalJSON --
 func (p *PayloadAttributesV2) MarshalJSON() ([]byte, error) {
+	withdrawals := p.Withdrawals
+	if withdrawals == nil {
```

Comment:
why was this added in ?

Reply:
presumably this fixes an issue where an empty withdrawals list results in a nil slice. You're probably wondering why this isn't needed in capella/develop though, cc @terencechain who is on the blame.